### PR TITLE
Update UI for example projects

### LINKED ISSUE #95: Update UI in Augury Examples Project
The UI for the examples in the [Augury Docs site](http://augury.rangle.io) hasn't been updated for quite some time. Examples should be updated to look more modern, and contain a more uniform design across all examples.

### PR DESCRIPTION
**Pull request by yallen011**
Updated ui for the example projects.
Closes #95

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -5,12 +5,28 @@
...
+.amber {
+  color: #ffd149;
+}
+
+.dark-amber-bg {
+  background: #c67100
+
+
+
+  ;
```
Comment (feeloor) [inline src/styles.css]: Can we remove the spacing here?

File: src/styles.css [hunk 2]
```diff
@@ -5,12 +5,28 @@
   font-family: "Roboto", sans-serif;
 }
 
-body {
+/* body {
   background: #f1e0ff;
-}
+} */
```
Comment (feeloor) [inline src/styles.css]: Is there a reason we want to remove this? If we are commenting it out can we just remove it completely instead?

File: src/styles.css (hunk 1 above)
Comment (yallen011) [reply src/styles.css]: fixed

**Comment by feeloor** [comment]
LGTM 👍

**Pull request merged**